barcodescanner/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,35 @@
+<resources>
+
+    <string name="barcode_scanner_fragment_reading_barcode_info">
```

Comment:
We should be able to customize those from outside the module.

I'm not sure this is viable, if I can add those Ids to the app and it would compile, I don't think so. We should check a viable way to do this.

Reply:
Actually it's quite simple, please add to the docs
https://stackoverflow.com/questions/4263259/replace-or-override-string-in-android-library-project